### PR TITLE
remove-number

### PR DESCRIPTION
**Pull request by koala-droid**
Sorry for being late,
I am not sure about the logic of the code and at the same time I am just confused about the appropriate methods to use.
The exercise for me is not explanatory enough, so I had to look up how other people handled that.  Yet, I am not fully sure I understand it.

Please, see if that makes sense to you.

#7

### CODE REVIEW

**Comment by Sayed94h** [comment]
In my opinion the logic is like this: 

1- Read the array (arrOfNumbers): use for or for...of loop
2- Compare each item in the array (arrOfNumbers) with the parameter (toRemove)
3- If item in the array (arrOfNumbers) is not strictly equal to the parameter (toRemove), then push the item to the shared variable (script-varaible.js which is the variable 'numbers').

```js
for(let item of arrOfNumbers) {

if (item !== toRemove) {
numbers.push(item);
}
}
```

This is what I think

File: scripts/user-actions/remove-number.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const removeNumber = (arrOfNumbers, toRemove) => {
   }
 
   // write the logic!
```
Comment (Sayed94h) [inline scripts/user-actions/remove-number.js:23]: I commented my opinion about the logic. So, you can test it once. I hope it would help.

Comment (oguzkarademir) [reply scripts/user-actions/remove-number.js:23]: I checked on js-tutor, it worked 👍

**Review by oguzkarademir — APPROVED** [review]

**Review by oguzkarademir — CHANGES_REQUESTED** [review]
Can you add your user story on `development-strategy.md`

File: scripts/user-actions/remove-number.js (hunk 1 above)
Comment (koala-droid) [reply scripts/user-actions/remove-number.js:23]: thank you, guys!

**Review by oguzkarademir — APPROVED** [review]

**Pull request merged**